packages/styles/combobox.css

```diff
@@ -111,8 +111,28 @@
   position: relative;
 }
 
+.Combobox .TextFieldWrapper {
+  flex-wrap: wrap;
+}
+
 .Combobox .TextFieldWrapper input[type='text'] {
   margin-right: 1.25rem;
+  flex-grow: 1;
+  width: unset;
+  min-width: var(--input-min-width);
+}
+
+.Combobox__selected-tag {
+  margin-block: var(--space-smallest);
+}
+
+.Combobox__selected-tag:first-of-type {
+  margin-left: var(--space-smallest);
+}
+
+.Combobox__selected-tag .TagButton__icon svg {
+  width: 16px;
+  height: 16px;
```

Comment:
I think instead of defining the size here, we may be able to just ensure we're using `--icon-size: 16px` scoped to the right css selector.